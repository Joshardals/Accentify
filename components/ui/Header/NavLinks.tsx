import { CiSearch } from "react-icons/ci";
import Link from "next/link";

const links = [
  { label: "New In", href: "#" },
  { label: "Shop", href: "#" },
  { label: "About Us", href: "#" },
  { label: "Search", href: "#", icon: <CiSearch size={20} /> },
];
export default function NavLinks() {
  return (
    <nav>
      <ul className="flex items-center space-x-10 linkList">
        {links?.map((link) => {
          const { label, href, icon } = link;
          return (
            <li key={label}>
              <Link
                href={href}
                className={`uppercase font-light ${
                  icon && "flex items-center space-x-1"
                }`}
              >
                {icon}
                <p> {label}</p>
              </Link>
            </li>
          );
        })}
      </ul>
    </nav>
  );
}
