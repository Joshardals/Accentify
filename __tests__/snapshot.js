import { render } from '@testing-library/react'
import Page from "@/app/(root)/page";
import NavLinks from "@/components/ui/Header/NavLinks";


it('renders homepage unchanged', () => {
    const { container } = render(<Page />)
    expect(container).toMatchSnapshot()
})

it(`renders navlinks unchanged`, () => {
    const { container } = render(<NavLinks />)
    expect(container).toMatchSnapshot()
    expect()
})