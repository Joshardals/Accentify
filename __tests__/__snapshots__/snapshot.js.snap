// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <main>
    <h1>
      home
    </h1>
  </main>
</div>
`;

exports[`renders navlinks unchanged 1`] = `
<div>
  <nav>
    <ul
      class="flex items-center space-x-10 linkList"
    >
      <li>
        <a
          class="uppercase font-light undefined"
          href="#"
        >
          <p>
             
            Shop
          </p>
        </a>
      </li>
      <li>
        <a
          class="uppercase font-light undefined"
          href="#"
        >
          <p>
             
            About Us
          </p>
        </a>
      </li>
      <li>
        <a
          class="uppercase font-light flex items-center space-x-1"
          href="#"
        >
          <svg
            fill="currentColor"
            height="20"
            id="search"
            stroke="currentColor"
            stroke-width="0"
            version="1.1"
            viewBox="0 0 24 24"
            width="20"
            x="0px"
            xmlns="http://www.w3.org/2000/svg"
            y="0px"
          >
            <g>
              <path
                d="M20.031,20.79c0.46,0.46,1.17-0.25,0.71-0.7l-3.75-3.76c1.27-1.41,2.04-3.27,2.04-5.31
		c0-4.39-3.57-7.96-7.96-7.96s-7.96,3.57-7.96,7.96c0,4.39,3.57,7.96,7.96,7.96c1.98,0,3.81-0.73,5.21-1.94L20.031,20.79z
		 M4.11,11.02c0-3.84,3.13-6.96,6.96-6.96c3.84,0,6.96,3.12,6.96,6.96c0,3.84-3.12,6.96-6.96,6.96C7.24,17.98,4.11,14.86,4.11,11.02
		z"
              />
            </g>
          </svg>
          <p>
             
            Search
          </p>
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
